app/Http/Controllers/UserController.php

```diff
@@ -241,21 +197,15 @@ public function resetPassword(ResetPasswordRequest $request)
     {
         $device = Auth::guard('api')->user();
 
-        if ($device && ($user =  $device->user)) {
-            if (Hash::check($request->oldPassword, $user->password)) {
-                $user->update([
-                    'password' => Hash::make($request->newPassword)
-                    ]);
-                $response ['code'] = '0001';
-            } else
-            $response ['code'] = '0021';
-        } else
-        $response = [
-        'code' => '0032',
-        ];
-        $response['message'] = $this->codeMessage->code($response['code']);
+        if (!($device && $user =  $device->user) )
+            return response(Helper::userResponse('0032'));
 
-        return response($response);
+        if (!Hash::check($request->oldPassword, $user->password))
+            return response(Helper::userResponse('0021'));
+         
+        $user->update(['password' => Hash::make($request->newPassword)]);
+              
+        return response(Helper::userResponse('0001'));
```

Comment:
lots of use of Helper class. won't it be better to make its instance for each file and use the same all times?